### PR TITLE
fix(core): remove ngcc postinstall migration

### PR DESCRIPTION
**Pull request by filipesilva**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
angular/angular-cli#16017

## What is the new behavior?
No ngcc added automatically to postinstall

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Other information
Partially address angular/angular-cli#16017

### CODE REVIEW

File: aio/content/guide/ivy.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ In the `angular.json` workspace configuration file, set the default build option
 
 Ivy applications can be built with libraries that were created with the View Engine compiler.
 This compatibility is provided by a tool known as the Angular compatibility compiler (`ngcc`).
-CLI commands run `ngcc` as needed, either after npm installation of dependencies or when performing an Angular build.
+CLI commands run `ngcc` as needed performing an Angular build.
+You can also run `ngcc` after npm installation for faster compilation as described in [Speeding up ngcc compilation](#speeding-up-ngcc-compilation).
```
Comment (jbogarthyde) [inline aio/content/guide/ivy.md]: The new section should not be separated from this, it's part of the same topic. How about making it a subsection?
Change this   to something like:

### Speed up compilation for large projects

To speed up compilation for a large project with many dependent libraries, you can use a post-install script to run `ngcc` in parallel. 

*move section content here*

File: aio/content/guide/ivy.md [hunk 2]
```diff
@@ -38,7 +38,8 @@ In the `angular.json` workspace configuration file, set the default build option
...
```
Comment (jbogarthyde) [inline aio/content/guide/ivy.md]: Make this a subsection as well :

### Maintaining library compatibility

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
Organizational suggestions

**Comment by filipesilva** [comment]
@jbogarthyde added your suggestions, and also added an alert box with a caveat to this approach, PTAL.

File: aio/content/guide/ivy.md [hunk 3]
```diff
@@ -38,7 +38,33 @@ In the `angular.json` workspace configuration file, set the default build option
 
 Ivy applications can be built with libraries that were created with the View Engine compiler.
 This compatibility is provided by a tool known as the Angular compatibility compiler (`ngcc`).
-CLI commands run `ngcc` as needed, either after npm installation of dependencies or when performing an Angular build.
+CLI commands run `ngcc` as needed performing an Angular build.
+
+{@a speeding-up-ngcc-compilation}
+## Speeding up ngcc compilation
```
Comment (jbogarthyde) [inline aio/content/guide/ivy.md]: Please make these subsections (h3)

File: aio/content/guide/ivy.md [hunk 4]
```diff
@@ -38,7 +38,33 @@ In the `angular.json` workspace configuration file, set the default build option
...
+<div class="alert is-important">
+
+The `postinstall` script will run on every installation of `node_modules`, including those performed by `ng update` and `ng add`.
+
+If you perform multiple installs in a row, this can end up being slower than letting Angular CLI run `ngcc` on builds.
+
+</div>
+
+{@a maintaining-library-compatibility}
+## Maintaining library compatibility
```
Comment (jbogarthyde) [inline aio/content/guide/ivy.md]: subsection (h3)

**Review by jbogarthyde — APPROVED** [review]
Please change heading levels - looks good otherwise.

**Comment by filipesilva** [comment]
Changed the headers, thanks for the review @jbogarthyde!

**Review by IgorMinar — APPROVED** [review]
I thought that we were going to make this migration optional (you'd have to invoke it via an explicit command) but I think the change to package.json is trivial enough that documenting it is sufficient.

thanks!

**Comment by IgorMinar** [comment]
merge-assistance: no g3 impact - we don't run this migration in g3

**Pull request closed without merging**